recipes/containers-sugar/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "containers-sugar" %}
+{% set version = "1.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/containers_sugar-{{ version }}.tar.gz
+  sha256: 28c9ed69d61a52016065af927f00eea147eb148ff383c313543da1e9514a15c6
+
+build:
+  noarch: python
```

Comment:
```suggestion
```

This is something that I always have problems in the conda recipe.

but normally if there is a skip, you cannot use noarch ... but if you remove noarch, you need to remove the pinning for python?? ... I always struggles with that XD

but for now just remove the noarch entry

https://github.com/conda-forge/staged-recipes/pull/24383#issuecomment-1787034899